<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="com.fullcycle.admin.catalogo.application.category.update.UpdateCategoryUseCaseTest" tests="5" skipped="0" failures="0" errors="0" timestamp="2024-02-08T13:30:18" hostname="glauber-Aspire-A315-41" time="0.032">
  <properties/>
  <testcase name="givenAValidCommand_whenGatewayThrowsRandomException_shouldReturnAException()" classname="com.fullcycle.admin.catalogo.application.category.update.UpdateCategoryUseCaseTest" time="0.007"/>
  <testcase name="givenAValidInactivateCommand_whenCallsUpdateCategory_shouldReturnInactiveCategoryId()" classname="com.fullcycle.admin.catalogo.application.category.update.UpdateCategoryUseCaseTest" time="0.006"/>
  <testcase name="givenACommandWithInvalidID_whenCallsUpdateCategory_shouldReturnNotFoundException()" classname="com.fullcycle.admin.catalogo.application.category.update.UpdateCategoryUseCaseTest" time="0.005"/>
  <testcase name="givenAValidCommand_whenCallsUpdateCategory_shouldReturnCategoryId()" classname="com.fullcycle.admin.catalogo.application.category.update.UpdateCategoryUseCaseTest" time="0.005"/>
  <testcase name="givenAInvalidName_whenCallsUpdateCategory_thenShouldReturnDomainException()" classname="com.fullcycle.admin.catalogo.application.category.update.UpdateCategoryUseCaseTest" time="0.004"/>
  <system-out><![CDATA[]]></system-out>
  <system-err><![CDATA[]]></system-err>
</testsuite>
